index.html

```diff
@@ -9,30 +9,46 @@
                 
 </head>
 <body>
+    <section id="main-page">
      <header class="nav-bar">
                 <a class="logo" href="#">Muhammad Nafees</a>
-                <nav class="menu" href="#"><img src="images/Union.png" alt="Menu bar"></nav>
+                <nav class="menu" href="#"><img id="menu-icon" src="images/menu.png" alt="Menu bar">
+                <div id="nav-menu">
+                    <a href="#">Portfolio</a>
+                    <a href="">About</a>
+                    <a href="">Contact</a>
+                    <a href=""><img src="images/mail-icon.png" alt=""></a>
+                </div></nav>
      </header>
         <section class="content">
+            <div class="main-text">
     <h1 class="my-name">Hey There.<br/> I'm Nafees</h1>
     <h3 class="profession">I’m a Software Developer</h3>
     <p class="i-can-do">I can help you build a product , <br /> feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
         don’t hesitate to contact me.</p>
+    </div>
         <ul class="links">
             <li><a title="GitHub" href="https://github.com/ngeelani48"><img src="images/github.png" alt="GitHub Profile"></a></li>
             <li><a title="Twitter" href="https://twitter.com/ngeelani48"><img src="images/twitter.png" alt="Twitter Profile"></a></li>
             <li><a title="Medium" href="https://medium.com/@engnrnafees"><img src="images/medium.png" alt="Medium Profile"></a></li>
             <li><a title="LinkedIn" href="https://www.linkedin.com/in/m-n-geelani-3446097275/"><img src="images/Linkedin icon.png" alt="LinkedIn Profile"></a></li>
             <li><a title="Angelist" href="#"><img src="images/Vector.png" alt="angelist Profile"></a></li>
-        </ul>
+        </ul>  
     </section>
+</section>
     <section class="project-section">
+        <div class="my-recent-works">
         <div class="my-work-title">
             <h1>My Recent Works</h1>
         </div>
         <div id="line"></div>
+    </div>
```

Comment:
- [ ] I think you have introduced this bug while implementing the changes asked by the previous reviewer, please make sure you follow the second design image to update yours

<img width="1221" alt="1" src="https://user-images.githubusercontent.com/77843050/213572837-0b98345b-46c0-447e-8373-5ba11a2694dc.png">

<img width="784" alt="2" src="https://user-images.githubusercontent.com/77843050/213573267-2e64d25b-7527-42e8-805f-8748a979d0cc.png">